pkgs/top-level/all-packages.nix

```
@@ -16341,6 +16341,8 @@ in
 
     facetimehd = callPackage ../os-specific/linux/facetimehd { };
 
+    tuxedo-keyboard = callPackage ../os-specific/linux/firmware/tuxedo-keyboard { };
```

```suggestion
    tuxedo-keyboard = callPackage ../os-specific/linux/tuxedo-keyboard { };
```
I was mistaken during our call, the other drivers are one level up. The firmware folder is for binary blobs. Don't forget to also move the file.